lib/audit/auditor_test.go

```diff
@@ -352,6 +352,309 @@ func TestAudit(t *testing.T) {
 }`,
 	}
 
+	// Regression test for the case where we are promoting
+	// bcdd5657b1edc1a2eb27356f33dd66b9400d4a084209c33461c7a7da0a32ebb3 (etcd
+	// tag "3.4.7-2"). It is crucial that we read fom *ALL* source registries,
+	// not just the first one that has a match for the destination registry
+	// "us.gcr.io/k8s-artifacts-prod" (because both manifests below promote to
+	// that same destination, but only one of them have the child images of
+	// 3.4.7-2).
+	manifests2 := []reg.Manifest{
```

Comment:
You could make this a lot more readable with a couple `makeFoo()` helpers that take the params you want to populate and produce these structs - save on a bunch of curlies and lines.